libs/math/include/math/TQuatHelpers.h

```diff
@@ -244,8 +244,9 @@ class TQuatFunctions {
     QUATERNION<T> MATH_PURE pow(const QUATERNION<T>& q, T a) {
         // could also be computed as: exp(a*log(q));
         const T nq(norm(q));
-        const T theta(a * std::acos(q.w / nq));
-        return std::pow(nq, a) * QUATERNION<T>(normalize(q.xyz) * std::sin(theta), std::cos(theta));
+        const T cos_theta = q.w / nq;
```

Comment:
where did the `a*` go?

`cos(a*acos(x)) != x`

Reply:
Ah, missed that. Thanks.